src/go/accessd/main.go

```diff
@@ -21,8 +21,8 @@ import (
 
 func main() {
 	lm := log.NewManager(zap.NewLogger(uber_zap.NewDevelopmentConfig()))
+	server.Start(lm.LoggerFor("server"))
 
-	if err := server.Start(lm.LoggerFor("server")); err != nil {
-		panic(err)
-	}
+	stopper := make(chan struct{})
+	<-stopper
```

Comment:
is this being used for anything right now?

Reply:
no use right now, but needs to be there since i made the grpc server start in a goroutine, so main() will exit without a stopper